pkg/baseline/packageIndex.json

```diff
@@ -22,15 +22,17 @@
         "4.3.0",
         "4.3.1",
         "4.4.0",
-        "4.4.1"
+        "4.4.1",
+        "4.4.2"
       ],
       "BaselineVersion": "4.4.1",
```

Comment:
Should this be updated too?

Reply:
If I recall correctly that just makes sure that during sync we at least restore that package version, I was going to update that once we actually had a 4.4.2 pkg on MyGet.